ibis/formats/pandas.py

```diff
@@ -64,7 +64,9 @@ def schema_from_pandas(schema):
     return sch.schema(ibis_types)
 
 
-def schema_from_pandas_dataframe(df: pd.DataFrame, schema=None):
+def schema_from_pandas_dataframe(
+    df: pd.DataFrame, schema=None, inference_function=infer_sequence_dtype
```

Comment:
Do we need to expose the optionality of `inference_function` here?

Reply:
You mean `_inference_function`?